spec/inputs/snmp/mib_spec.rb

```diff
@@ -17,30 +17,23 @@ module LogStash
     end
 
     it "should produce 0 warning when first adding a mib path" do
-      warnings = subject.add_mib_path(rfc1213_mib)
-      expect(warnings.size).to eq(0)
+      expect_any_instance_of(SnmpMib::Tree).to receive(:logger).never.and_call_original
+      subject.add_mib_path(rfc1213_mib)
     end
 
     it "should produce 0 warning when adding same keys and values" do
-      warnings = subject.add_mib_path(rfc1213_mib)
-      expect(warnings.size).to eq(0)
-      warnings = subject.add_mib_path(rfc1213_mib)
-      expect(warnings.size).to eq(0)
-    end
+      expect_any_instance_of(SnmpMib::Tree).to receive(:logger).never.and_call_original
```

Comment:
I'm curious, do you need the and_call_original when prefixed with `.never.`?
Not material to the review though.

Reply:
true that :P. will leave it for now, these specs are relatively stupids to rely on the logger calls to verify some of the logic and I am planning on refactoring them, these changes were a quick way to make it work using the "same" idea as the old `warnings` array and this `never.and_call_original` is the result of a copy&paste :P